entries/g/geforce.com.json

```diff
@@ -1,13 +1,19 @@
 {
   "GeForce (Nvidia)": {
     "domain": "geforce.com",
-    "url": "https://www.geforce.com/",
-    "contact": {
-      "facebook": "NVIDIAGeForce",
-      "twitter": "NVIDIAGeForce"
-    },
+    "additional-domains": [
+      "nvidia.com",
+      "login.nvgs.nvidia.com"
```

Comment:
Is this still useful when `nvidia.com` is already listed?
```suggestion
```

Reply:
I thought that it could potentially be useful for password manager support. I know that bitwarden doesn't check subdomains for 2FA support and 1Password only does it 1 subdomain deep: 
![image](https://user-images.githubusercontent.com/20560218/127916383-f2964865-9b27-487f-88d9-9d27744ac18a.png)
![image](https://user-images.githubusercontent.com/20560218/127916410-524e7df8-f246-4691-8e7e-1f03d57a5473.png)